homeassistant/components/transmission/sensor.py

```diff
@@ -22,6 +28,70 @@
 )
 
 
+@dataclass
+class TransmissionSensorEntityDescription(SensorEntityDescription):
+    """Describe Transmission sensor entity."""
+
+    name_suffix: str | None = None
+    sub_type: str | None = None
+
+
+SPEED_SENSOR_DESCRIPTIONS = [
+    TransmissionSensorEntityDescription(
+        key="download",
+        name_suffix="Down Speed",
```

Comment:
Maybe use the new style entity naming directly?
<https://developers.home-assistant.io/docs/core/entity#entity-naming>

Reply:
unresolved this comment, as it  has not been addressed.

Reply:
Not sure what you mean by this. I'm using new style entity naming and got rid of `name_suffix` in favor of `name`. What else do you expect here?